dev-manuals/test-plans/lecturer-interface/u-lecturer-interface-17-minigames-maps.md

```diff
@@ -0,0 +1,34 @@
+# Lecturer-Interface: Minigame maps (`u.lecturer-interface-17`)
+
+
+Version: V1.0, 17.08.2022 \
+Author: Florian Wüst, Gilian Rehm
+
+## Description
+
+The maps showing the positions of all minigames of an area are correct.
+
+## Precondition
+
+All minigames in the area are configurated and a map is created and displayed in the lecturer interface.
+
+## Postcondition
+
+N/A
+
+## Typical procedure
+
+1. All world maps show the correct minigame positions
+2. All dungeon maps show the correct minigame positions
```

Comment:
```suggestion
1. For all world maps, the minigame spots displayed are up to date with the current state of the world in the overworld
2. For all dungeon maps, the minigame spots displayed are up to date with the current state of the dungeon in the overworld
```
Hmm, I just noticed: We are going to get problems with different versions:
We cannot simply link `main` from the lecturer interface, because if you then have an old lecturer interface, and the overworld has been changed in the meantime, you end up with an incorrect map...